### PR TITLE
add build_ffmpeg options

### PR DESCRIPTION
**Pull request by cdeepali**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,6 +137,13 @@ The system where FIPS compliant packages are to be installed must have Redhat's
 
 Once FIPS complaint Open-CE packages are installed in a conda environment, ensure that the conda environment contains  `openssl` and `cryptography` also from the same conda channel.
 
+### FFMPEG with reduced codecs and openssl
+#### Building packages with OpenCE ffmpeg
+
+One can build Open-CE packages with ffmpeg by providing `--build-ffmpeg` option to `open-ce build env` command. When this option is provided, Open-CE Builder builds [`envs/ffmpeg-env.yaml`](https://github.com/open-ce/open-ce/blob/main/envs/ffmpeg-env.yaml) internally and then builds the provided Open-CE environment file for which the build is requested.
+
+FIPS compliant packages are built by default with `ffmpeg-env.yaml`.
```
Comment (alexlang74) [inline README.md:145]: That means that `--build-ffmpeg True` automatically sets `--fips True` , correct?

**Review by alexlang74 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cdeepali) [reply README.md:145]: I think there is a copy paste mistake in that line. I will correct it in a while.

File: open_ce/inputs.py [hunk 2]
```diff
@@ -408,7 +413,11 @@ def parse_args(parser, arg_strings=None):
     args = parser.parse_args(arg_strings)
     _create_env_config_paths(args)
 
-    _check_and_create_fips_packages(args, arg_strings)
+    #--fips and --build_ffmpeg together is not supported
```
Comment (alexlang74) [inline open_ce/inputs.py:416]: why?  I thought when you do `--build_ffmpeg` , we also build fips (i.e., the same as the `--fips`) option?

**Review by alexlang74 — APPROVED** [review]

File: open_ce/inputs.py (hunk 2 above)
Comment (cdeepali) [reply open_ce/inputs.py:416]: when we do` --fips` we are invoking build of `openssl-env.yaml` and `openssl-env.yaml` also contains listing of fips complaint ffmpeg feedstock. 
So `build_ffmpeg` is not needed with `--fips`. The ffmpeg feedstock will be built by default when we provide `--fips`.

**Review by npanpaliya — APPROVED** [review]

**Review by npanpaliya — APPROVED** [review]

**Pull request merged**